lib/simp/metadata/buildinfo.rb

```diff
@@ -30,8 +30,8 @@ def fetch_data
 
       def method_defaults
         {
-          'rpm' => {
-            'build_method' => 'simp-core'
+            :rpm => {
+                :build_method => 'simp-core'
```

Comment:
It's possible I'm being tricked by the diff syntax, but I believe you have this indented by 4 spaces, not 2.